packages/asc-ui/src/internals/CenteredElement/CenteredElement.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react'
+import styled from '../../styled-components'
+
+const CenteredElementStyle = styled.div`
+  display: flex;
+  height: 100%;
+  & > div {
+    align-self: center;
+    flex-grow: 1;
+    text-align: center;
+  }
+`
+
+const CenteredElement: React.FC<{}> = ({ children }) => (
+  <CenteredElementStyle>
+    <div>{children}</div>
```

Comment:
Could this also be done without rendering another element to the DOM?

Reply:
no